app/src/main/res/layout/tip_carousel_item.xml

```diff
@@ -5,11 +5,11 @@
 <com.google.android.material.textview.MaterialTextView xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:gravity="center"
+    android:layout_marginBottom="16dp"
+    android:gravity="center|bottom"
     android:lineSpacingExtra="7sp"
+    android:lines="5"
```

Comment:
Do we want to have **exactly** this many lines, or to just **allow** this many lines?

Reply:
To allow. Probably you will suggest to use maxLines instead of lines but I tried that and it didn't work. Probably because we are using layout_weight in fragment_urlinput2

Reply:
No, it's because the first item displayed will give the height of the next ones in the current adapter used. But I guess it would be more work to make it adapt to each item's height so the workaround of adding bottom gravity should be enough for this layout in its present state.